src/libs/events.test.ts

```diff
@@ -37,7 +37,20 @@ describe('extractWorkspaceDetails', () => {
       workspaceName: defaultGoogleWorkspace.workspace.name,
       workspaceNamespace: defaultGoogleWorkspace.workspace.namespace,
       cloudPlatform: 'GCP',
-      hasProtectedData: undefined,
+      hasProtectedData: false,
+    });
+  });
+
+  it('Determine hasProtectedData based on workspace.policies', () => {
```

Comment:
At some point we probably want to change the eventing to consider GCP workspaces with the special bucket prefix as protected too. But for now I left the code as-is.